massgov/mayflower
# Feature/dp 13864  details clickable icons

## Description
This ticket adds the ability for icons to be clickable in the sticky table of contents linked items.

## Related Issue / Ticket

- JIRA: DP-DP-13864

## Steps to Test
*Go Here*
https://mayflower.digital.mass.gov/b/feature/DP-13864--details-clickable-icons/patterns/05-pages-information-details-table-of-contents/05-pages-information-details-table-of-contents.html?1560180385177

*Do This*
- Hover over the green arrow icons in the Table of Contents section.
- Click on one of the green icons

*You Should See*
- A cursor showing over the green arrow icon upon hover
- Upon clicking the green arrow icon, it should have the ability to click and take a user to the same destination as the paired link will
